src/bilder/images/edxapp/files/edx_web_playbook.yml

```diff
@@ -0,0 +1,26 @@
+---
+- name: Deploy edxapp
+  hosts: all
+  become: true
+  gather_facts: true
+  vars:
+    serial_count: 1
+    CLUSTER_NAME: edxapp
+    NGINX_ENABLE_SSL: false
+    COMMON_CONFIG_NO_LOGGING: false
+    # yamllint disable-line rule:line-length
+    EDXAPP_EDXAPP_SECRET_KEY: this-is-just-a-temporary-placeholder # pragma: allowlist secret
+    CREATE_SERVICE_WORKER_USERS: false
+  serial: '{{ serial_count }}'
+  roles:
+  - role: automated
+    AUTOMATED_USERS: '{{ EDXAPP_AUTOMATED_USERS | default({}) }}'
+  - role: nginx
+    nginx_sites:
+    - lms
+    - cms
+    nginx_default_sites: '{{ EDXAPP_NGINX_DEFAULT_SITES }}'
+    nginx_extra_sites: '{{ NGINX_EDXAPP_EXTRA_SITES }}'
+    nginx_extra_configs: '{{ NGINX_EDXAPP_EXTRA_CONFIGS }}'
+    nginx_skip_enable_sites: '{{ EDXAPP_NGINX_SKIP_ENABLE_SITES }}'
```

Comment:
Should we stick to one standard as in either `EDXAPP_NGINX` or `NGINX_EDAPP`?

Reply:
This is a copy/paste from the edX configuration repo and I just deleted the roles that we don't want, so the values here are just what the repo there is expecting to be able to pull in.